modules/oe_theme_content_publication/src/Plugin/PageHeaderMetadata/PublicationContentType.php

```diff
@@ -75,15 +76,30 @@ public function applies(): bool {
    * {@inheritdoc}
    */
   public function getMetadata(): array {
-    $metadata = parent::getMetadata();
-
     $node = $this->getNode();
-    $timestamp = $node->get('oe_publication_date')->date->getTimestamp();
-    $metadata['metas'] = [
-      $this->dateFormatter->format($timestamp, 'oe_theme_publication_date'),
-    ];
+
+    $metadata = parent::getMetadata();
+    $metadata['metas'][] = $this->getSeparatedSkosMeta($node->get('oe_publication_type'));
 
     return $metadata;
   }
 
+  /**
+   * Format a list of SKOS references into a comma separated string.
+   *
+   * @param \Drupal\rdf_skos\Plugin\Field\SkosConceptReferenceFieldItemList $items
+   *   Field item list object.
+   *
+   * @return string
+   *   Comma separated string.
+   */
+  protected function getSeparatedSkosMeta(SkosConceptReferenceFieldItemList $items): string {
```

Comment:
We could improve this method by adding the separator as parameter and implement it in a base class. It's already used in NewsContentType too but with a different parameter.